beater/server.go

```diff
@@ -64,7 +67,8 @@ func newServer(config Config, report reporter) *http.Server {
 }
 
 func run(server *http.Server, ssl *SSLConfig) error {
-	logp.Info("starting apm-server! Hit CTRL-C to stop it.")
+	logp.Info("Starting apm-server! Hit CTRL-C to stop it.")
```

Comment:
Going here from lower to upper case (which I like) is crashing all the system tests ..

Reply:
Thanks! I'll fix this in a separate PR